user-config-example.yml

```diff
@@ -0,0 +1,11 @@
+defaults:
+  aws:
+    accessKeyId:        '...'
+    secretAccessKey:    '...'
+    region:             us-west-2
+  azure:
+    accountName:        jungle
```

Comment:
YAY, remove the `accountKey` entry... no need... you can easily get jungle access using tc credentials..